### PR TITLE
Add tiles_scroll_perf__timeline_summary to devicelab

### PR DESCRIPTION
**Pull request by liyuqian**

### CODE REVIEW

File: dev/devicelab/manifest.yaml [hunk 1]
```diff
@@ -106,6 +106,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  tiles_scroll_perf__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
```
Comment (yjbanov) [inline dev/devicelab/manifest.yaml]: Unless you are testing something Mac-specific I'd use "linux/android". Our Linux agents are much more reliable than Macs.

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
LGTM with one comment.

lgtm

File: dev/devicelab/manifest.yaml (hunk 1 above)
Comment (liyuqian) [reply dev/devicelab/manifest.yaml]: Done. I've also changed complex_layout_perf to linux/android.

File: dev/devicelab/manifest.yaml [hunk 2]
```diff
@@ -104,7 +104,13 @@ tasks:
...
       Android.
     stage: devicelab
-    required_agent_capabilities: ["mac/android"]
+    required_agent_capabilities: ["linux/android"]
+
+  tiles_scroll_perf__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
```
Comment (Hixie) [inline dev/devicelab/manifest.yaml]: linux/android is a bit overloaded compared to mac/android, might be worth putting these on mac/android instead.

Please make sure to set flaky:true when first landing a new test.

Comment (yjbanov) [reply dev/devicelab/manifest.yaml]: Makes sense. We could get more linux/android agents too.

**Pull request merged**